src/components/admin/form-components/validated-fields.jsx

```diff
@@ -17,6 +17,15 @@ export const cannotNull = value => {
   return null;
 };
 
+export const hasVarCharMax = value => {
+  if (value.length > 150) {
+    return 'The database restricts this field to a maximum value of 150 characters.';
+  }
+  return null;
+};
+
 export const ValidatedHttpsUrlField = withValidate(hasHttps)(TextField);
 
-export const RequiredTextField = withValidate(cannotNull)(TextField);
+export const RequiredTextField = withValidate(cannotNull, hasVarCharMax)(
```

Comment:
Okay so this is where you implement the max length, but the name doesn't really reflect that it's bounded. I guess we're also allowing them to type whatever they want, but then the validation should stop them from saving I guess, I think I actually quite like that. Especially since I for example implemented markdown for the staff (I nearly said authors there, but corrected myself as I'm renaming it haha!) profiles where the length of the output can become shorter when you add a character and therefore limiting them from typing over could actually be incorrect.

The name should reflect that it's composing two validators though, no?

Reply:
The bounded has nothing to do with the max length.